scripts/globals/mobskills/proboscis_shower.lua

```diff
@@ -0,0 +1,26 @@
+---------------------------------------------
+-- Proboscis Shower
+-- Description: Restores HP.
+---------------------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    return 0
+end
+
+function onMobWeaponSkill(target, mob, skill)
+    local potency = skill:getParam()
+
```

Comment:
looks like pure copy pasta. at least comment where its from and that we don't have a proper formula.